### PR TITLE
created email field optional / Issue #35

### LINKED ISSUE #35: Make the e-mail field optional
Currently, the email field is mandatory, even if the user doesn't want to share his/her email. I don't want the e-mail field to appear at all, if the user doesn't want the update on e-mail.

### PR DESCRIPTION
**Pull request by BhuwanChandra**
closes #35

### CODE REVIEW

File: components/Books.js [hunk 1]
```diff
@@ -53,9 +53,14 @@ export default class Books extends React.Component {
       loader: true
     });
 
+    let email = "";
```
Comment (rkpattnaik780) [inline components/Books.js]: Logically the way you have handled email is absolutely correct but it seems there's slight compromise with the readability, what if you check for email and build the url inside the cases itself?

**Comment by BhuwanChandra** [comment]
Ok I will do the required changes

**Comment by rajat2502** [comment]
@BhuwanChandra Please squash all the commits into a single commit so that we can merge it!

**Comment by BhuwanChandra** [comment]
> @BhuwanChandra Please squash all the commits into a single commit so that we can merge it!

Done!

**Pull request merged**